selectors/emojis.ts

```diff
@@ -50,12 +76,16 @@ export function isCustomEmojiEnabled(state: GlobalState) {
 export const getOneClickReactionEmojis = createSelector(
     'getOneClickReactionEmojis',
     getEmojiMap,
-    getRecentEmojis,
-    (emojiMap, recentEmojis) => {
+    getRecentEmojisNames,
+    (emojiMap, recentEmojis: RecentEmojiData[] | string[]) => {
```

Comment:
Which of those types is now used for `recentEmojis`? I assume it's either one or the other at this point

Reply:
nice catch. I missed to update types here. have fixed it